### PR TITLE
Add string tables to be able to modify the strings of new languages

### PR DESCRIPTION
**Pull request by fienestar**

### CODE REVIEW

File: source/playlunky/mod/string_merge.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ bool StringMerger::RegisterOutdatedStringTable(std::string_view table)
     {
         return false;
     }
-    if (string_table > 8)
+    if (string_table > 12)
```
Comment (gmjosack) [inline source/playlunky/mod/string_merge.cpp:22]: should be 13 right?

Comment (fienestar) [reply source/playlunky/mod/string_merge.cpp:22]: Is there string13.str file? I didn't find it in Spellunky 2\Mods\Extracted

Comment (gmjosack) [reply source/playlunky/mod/string_merge.cpp:22]: It's 0 indexed

Comment (fienestar) [reply source/playlunky/mod/string_merge.cpp:22]: There is strings00 .. strings12 so string_table over 12 is invalid
checked value of string_table is 12 when file is strings12.str

Comment (gmjosack) [reply source/playlunky/mod/string_merge.cpp:22]: I'm going off of the original diff where it was >8 with strings07. will wait for malacath to comment though as i'm less familiar with this code

Comment (Malacath-92) [reply source/playlunky/mod/string_merge.cpp:22]: Looks to me like there was a bug in the old code. I won't be able to test this until tomorrow unfortunately, but it looks sane to me. If any of you wanna rest strings12 we can merge also today.

**Pull request merged**